File: Modul 2/Soal 5/PRAK205-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
#include <math.h>

int main () {
    float sisi_a, sisi_b, sisi_miring, alas, tinggi, keliling, luas;

    printf("Masukkan Panjang Sisi A: ");
    scanf("%f", &sisi_a);
    printf("Masukkan Panjang Sisi B: ");
    scanf("%f", &sisi_b);

    tinggi = sisi_a;
    sisi_miring = sisi_b;
    alas = sqrt(sisi_miring* sisi_miring - tinggi * tinggi);
    keliling = alas + tinggi + sisi_miring;
    luas = 0.5 * alas * tinggi;

    printf("Alas =  %.0f cm \n", alas);
    printf("Tinggi = %.0f cm \n", tinggi);
    printf("Keliling = %.0f cm \n", keliling);
    printf("Luas =  %.0f cm^2 \n", luas);

    return 0;
}

File: Modul 2/Soal 5/PRAK205-2410817220018-ANGGRAENI DWI ZAHRA.py
import math

sisi_a = float(input ("Masukkan Panjang Sisi A : "))
sisi_b = float(input ("Masukkan Panjang Sisi B : "))

tinggi = sisi_a
sisi_miring = sisi_b
alas = math.sqrt(sisi_miring**2 - tinggi**2)
keliling = alas + tinggi + sisi_miring
luas = 0.5 * alas * tinggi

print(f"Alas =  {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas =  {luas:.0f} cm^2")
